python/pyspark/sql/dataframe.py

```diff
@@ -1393,6 +1393,16 @@ def replace(self, to_replace, value=None, subset=None):
         |null|  null| null|
         +----+------+-----+
 
+        >>> df4.na.replace('Alice', None).show()
```

Comment:
Looks like now we allow something like `df4.na.replace('Alice').show()`. We're better add it here.